app/actors/hyku_addons/actors/member_collection_from_admin_set_actor.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module HykuAddons
+  module Actors
+    class MemberCollectionFromAdminSetActor < Hyrax::Actors::AbstractActor
+      def create(env)
+        ensure_collection!(env) if enabled? && correct_permissions?(env)
+
+        next_actor.create(env)
+      end
+
+      # We shouldn't be doing anything on update as the collection will be assigned on create
+      delegate :update, to: :next_actor
+
+      protected
+
+        def ensure_collection!(env)
+          admin_set_id = env.attributes.dig("admin_set_id")
+
+          return unless admin_set_id.present?
+
+          title = AdminSet.find(admin_set_id).title.first
+          collection = Collection.where(title: title)
```

Comment:
I think you can merge this line with the next. Also, `where` returns a collection proxy rather than a single instead like `find_by` would. It will work fine as long as there is a unique validation on the collection titles, but it is worth mentioning.

Reply:
I did try both `find_by` isn't in use because its `ActiveFedora` not `ActiveRecord` and using `where` stops me from having to deal with the `nil` errors if an `AdminSet` isn't found.